pkg/kubernetes/scripts/main.js

```diff
@@ -44,6 +44,7 @@
     require('./nodes');
     require('./topology');
     require('./volumes');
+    require('./virtual-machines.es6');
```

Comment:
Can we stick to standard js in the module? Or is there a reason we need to use es6?

Reply:
done